examples/05_vol_geom_wasm_threejs/index.html

```diff
@@ -52,7 +52,7 @@
 	var vologram = new Object();   // Stores state about the current vologram.
 	vologram.header_url = "./calif/header.vols";
 	vologram.sequence_url = "./calif/sequence_0.vols";
-	vologram.video_url = "./calif/output.mp4";
+	vologram.video_url = "./calif/output_720.mp4";
```

Comment:
i think this is correct @patrick-volograms ?

Reply:
I changed the structure of the examples in my last PR (sorry, forgot to tell you). I have all the examples referencing the same vologram in a different folder now.